src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+wall.addEventListener('click', event => {
+  let xCoordinate = event.offsetX - spider.offsetWidth / 2;
+  let yCoordinate = event.offsetY - spider.offsetHeight / 2;
+
+  xCoordinate = checkBorders(
+    xCoordinate,
+    spider.offsetWidth,
+    wall.clientWidth
+  );
+
+  yCoordinate = checkBorders(
+    yCoordinate,
+    spider.offsetHeight,
+    wall.clientHeight
+  );
+
+  spider.style.left = xCoordinate + 'px';
+  spider.style.top = yCoordinate + 'px';
+
```

Comment:
one small thing to enhance - instead of assigning the function return value to the same variable, let's put it right in spider.style.left and spider.style.top
+ it's better to create a variable for spider.offsetWidth (you have already used it in 4 places)

Reply:
I use spider.offsetWidth 2 times not 4 and I see no point in creating a variable for this